bigbluebutton-html5/imports/ui/components/join-handler/guest-wait/component.tsx

```
@@ -173,26 +183,42 @@ const GuestWait: React.FC<GuestWaitProps> = (props) => {
           <p aria-live="polite">{positionMessage}</p>
         </Styled.Position>
+        {hasCustomMessage && (
+          <Styled.MessageContainer>
+            <Styled.MessageLabel>
+              {intl.formatMessage(intlMessages.messageFromHost)}
+            </Styled.MessageLabel>
+            <Styled.MessageText
+              aria-live="polite"
+              data-test="guestMessage"
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={{ __html: message }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)